ROLES.md

```diff
@@ -241,76 +202,33 @@ are responsible for the overall technical health of their functional area.
 
 ### Requirements
 
-Getting to be a tech lead of an existing working group:
-
-- Recognized as having expertise in the group’s subject matter.
-
-- Approver for a relevant part of the codebase for at least 3 months.
-
-- Member for at least 6 months.
-
-- Should have reviewed or contributed to substantial PRs to the codebase.
-
-- Should be a `reviewer` in the OWNERS file.
-
-- Sponsored by the technical oversight committee.
-
-Additional requirements for leads of a new working group:
-
-- Originally authored or contributed major functionality to the group's area.
-
-### Responsibilities and privileges
-
-The following apply to the area / component for which one would be an owner.
-
-- Design/proposal approval authority over the area / component, though
-  escalation to the technical oversight committee is possible.
-
-- Technical review of [feature tracks](./mechanics/FEATURE-TRACKS.md).
-
-- Perform issue triage on GitHub.
-
-- Apply/remove/create/delete GitHub labels and milestones.
-
-- Write access to repo (assign issues/PRs, add/remove labels and milestones,
-  edit issues and PRs, edit wiki, create/delete labels and milestones).
-
-- Capable of directly applying lgtm + approve labels for any PR.
-
-  - Expected to respect OWNERS files approvals and use
-    [standard procedure for merging code](./REVIEWING.md#merging-prs).
-
-- Expected to work to holistically maintain the health of the project through:
+For existing working groups:
 
-  - Reviewing PRs.
+- Recognized as having expertise in the group’s subject matter
 
-  - Fixing bugs.
+- Approver for a relevant part of the codebase for at least 3 months
```

Comment:
Again, -1 on specific cooling off periods here.

Reply:
```suggestion
- Approver for a relevant part of the codebase
```
Just remove the timeframe?

Reply:
LGTM!